lib/rules/web/admin_console/4.2/common_ui_elements.xyaml

```diff
@@ -80,6 +80,16 @@ check_button_missing:
     selector:
       <<: *button_with_text
     missing: true
+click_button_title:
+  element:
+    selector:
+      xpath: //button[@title='<button_title>']
+    op: click
+check_button_title:
+  element:
+    selector:
+      xpath: //button[@title='<button_title>']
+    op: click
```

Comment:
any difference between `check_button_title` and `click_button_title` ?  Also is it different from `click_button_text` ?

Reply:
check_button_title just make sure button with the specific tile exists
click_button_title click button with specific title to perform an action
click_button_text locate the button with text and click to perform an action
Why  I locate button with title not with existing click_button_text? because the button here has no text, it's icon and only has title.